subsys/modem/modem_cmux.c

```diff
@@ -498,6 +513,31 @@ static void modem_cmux_acknowledge_received_frame(struct modem_cmux *cmux)
 	}
 }
 
+static void modem_cmux_send_psc(struct modem_cmux *cmux)
+{
+	struct modem_cmux_frame frame = {
+		.dlci_address = 0,
+		.cr = cmux->initiator,
+		.pf = true,
+		.type = MODEM_CMUX_FRAME_TYPE_UIH,
+		.data = (void *) &(struct modem_cmux_command) {
+			.type = {
+				.ea = 1,
+				.cr = 1,
+				.value = MODEM_CMUX_COMMAND_PSC,
+			},
+			.length = {
+				.ea = 1,
+				.value = 0,
+			},
+		},
```

Comment:
This would need an _encode API right?

Reply:
Yes, correct. This is relying on non-standard compiler behaviour.
Same as you said in the MSC command parsing.

I'll start reworking.